docs/extensibility/extend-rest-api-update-cart.md

```diff
@@ -0,0 +1,90 @@
+# Updating the cart with the Store API
```

Comment:
To keep consistency with the rest of our codebase, can we frame this doc as:
- Problem: What problem are we seeing here, this should be fairly easy to do since we only create new points After an existing problem.
- Solution: Describes the solution, can reference other documents.
- Basic Usage: The most minimal solution that can do the job.
- Things to consider: Any gotchas or things that people might need to know.
- API Definition: Full API docs in a table, see other documents.
- Putting it all together: A full example touching on all options that can be copy-pasted.

Reply:
Thanks Nadir, I've updated the document now to follow this format. I feel like there's going to be way too much to cover in the `Putting it all together` section, and a lot of irrelevant information not relating to this part of the API. I did not want to go into too much detail about the things not directly related to this, so I've omitted some parts that are covered in other documents, and also not gone into detail about the implementation of my example methods.

I also feel like there's quite a bit of overlap and duplication of the `Basic usage` and `Putting it all together` section, but this is probably OK.